tests/integration_tests/README.md

```diff
@@ -0,0 +1,18 @@
+# Integration tests
+
+
+## Requirements
+
+- Node
+- Go
+- Dep
+- Make
```

Comment:
Do you actually need Node, Dep, and Make to run these tests?

Reply:
Good catch! These are no longer needed as I'm using gomods

Reply:
I guess Node is required for `serverless`

Reply:
I think dep is the only one you're not using here.

Reply:
I believe no Make is being used @DarcyRaynerDD